src/Microsoft.VisualStudio.Mac.LanguageServices.Razor/DefaultVisualStudioWorkspaceAccessor.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.ComponentModel.Composition;
+using System.Linq;
+using Microsoft.CodeAnalysis;
+using Microsoft.VisualStudio.Editor.Razor;
+using MonoDevelop.Ide.TypeSystem;
+
+namespace Microsoft.VisualStudio.Mac.LanguageServices.Razor
+{
+    [System.Composition.Shared]
+    [Export(typeof(VisualStudioWorkspaceAccessor))]
+    internal class DefaultVisualStudioWorkspaceAccessor : VisualStudioWorkspaceAccessor
+    {
+        public DefaultVisualStudioWorkspaceAccessor()
+        {
+            Workspace = TypeSystemService.AllWorkspaces.First();
```

Comment:
Is my expectation that there will always be at least 1 workspace and that the first is the one we care about (we want registered Roslyn services)?

Reply:
Try http://source.monodevelop.com/#MonoDevelop.Ide/MonoDevelop.Ide.TypeSystem/TypeSystemService_WorkspaceHandling.cs,f1acb7fbdbce0331